multiversx_sdk/core/proto/transaction_pb2.pyi

```diff
@@ -1,39 +1,183 @@
+# -*- coding: utf-8 -*-
```

Comment:
Previously, the file was much shorter. 

Maybe we can check previous generation commands vs. current generation command.